source/_patterns/04-components/captioned-image/_captioned-image.scss

```diff
@@ -0,0 +1,27 @@
+// @file
+// Component: Captioned Image
+
+.captioned-image {
+  @media not speech {
+    display: table;
+    figcaption {
+      caption-side: bottom;
+      display: table-caption;
+    }
```

Comment:
I think there should be an empty line between `display:table;` and `figcaption {`, but I’m not sure if that’s documented in our coding standards.

Any reason not to add a `captioned-image__caption` class on the `figcaption` element and use the class selector here instead?